src/Symfony/Component/Routing/Loader/AnnotationClassLoader.php

```diff
@@ -265,5 +351,7 @@ protected function createRoute($path, $defaults, $requirements, $options, $host,
         return new Route($path, $defaults, $requirements, $options, $host, $schemes, $methods, $condition);
     }
 
-    abstract protected function configureRoute(Route $route, \ReflectionClass $class, \ReflectionMethod $method, $annot);
+    protected function configureRoute(Route $route, ReflectionClass $class, ReflectionMethod $method, $annot)
```

Comment:
why making it a concrete method ?

Reply:
The tests for this class were testing through a partial mock, which is not a good way of testing it. In order to make the entire class concrete this method, which is merely a "hook", needs to be concrete too.

Reply:
Could we keep it protected, and replace the mock by a child class? We're doing so in other places, with the class in the same file as the tests, below them.

Reply:
@nicolas-grekas do you mean keep it abstract?

Reply:
yes if possible, so that implementers are forced to implement this from beginning